test/pymmh3_test.py

```diff
@@ -28,7 +28,8 @@ def test_32bit_basic_string( self ):
         with open( os.path.join( file_dir, 'pg1260.txt' ), 'rb' ) as test_file:
             for l in test_file.readlines():
                 s = solution[l]
-                r = pymmh3.hash( l )
+                # A negative value in python has an infinite number of leading 1's in binary unlike c++ where only MSB is 1 in 2's complement form.
+                r = pymmh3.hash( l ) & 0xFFFFFFFF
```

Comment:
I did regenerate solution files. They stayed the same.
However comparing with pymmh3 is a good idea :)